machineconfiguration/v1/types.go

```diff
@@ -148,7 +148,7 @@ type ControllerConfigSpec struct {
 
 	// ipFamilies indicates the IP families in use by the cluster network
 	// +required
-	IPFamilies IPFamiliesType `json:"ipFamilies"`
+	IPFamilies configv1.IPFamiliesType `json:"ipFamilies"`
```

Comment:
Do we need to do this? We tend to prefer local types over importing from other packages

Reply:
IPFamiliesType was originally defined here : https://github.com/openshift/api/blob/master/machineconfiguration/v1/types.go#L177-L185.

Instead of redefining IPFamilies for use within types_infrastructure.go, i moved it to a common location in config/v1/types.go

Reply:
I do not envision IP family types to diverge. But, let me try again by separating the APIs.